lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -172,5 +174,19 @@ public function basic_validation( $old_options, $input ) {
 		return $input;
 	}
 
+	public function render_wpml_support() {
```

Comment:
Can we check for a core WPML function or class with `function_exists()` or `class_exists()` to determine whether this setting should render?

Reply:
Yes it's possible with `if ( function_exists('icl_object_id') ) { }`